src/Terminal/commands/download.ts

```diff
@@ -25,7 +25,7 @@ export function exportScripts(pattern: string, server: BaseServer, currDir = roo
     .then((content: Blob) => downloadContentAsFile(content, filename))
     .catch((error) => {
       console.error(error);
-      Terminal.error(`Cannot compress scripts with pattern ${pattern} on ${server.hostname}. Error: ${String(error)}`);
+      Terminal.error(`Cannot compress scripts with pattern ${pattern} on ${server.hostname}. Error: ${error}.`);
```

Comment:
Small nit: This seems like a revert, but you added a period. Not a huge deal, but please check that **either all or none** of the similar cases in these files have periods in the message.